### ISSUE TITLE
Missing source-mapping when using autonumeric2

### ISSUE THREAD

**Issue by dwilkie** [randoum/autonumeric-rails#11]
I get the following error after upgrading to autonumeric2

```
ActionController::RoutingError (No route matches [GET] "/assets/autoNumeric.min.js.map")
```

It's due to the following line in `autoNumeric-2.0.js`
```js
//# sourceMappingURL=autoNumeric.min.js.map
```

**Comment by randoum** [comment]
@dwilkie could you test with master branch and tell me if its fixed?

**Comment by dwilkie** [comment]
@randoum awsome https://github.com/randoum/autonumeric-rails/commit/209af266a055465a991aeb228a154f566ed230f3 fixes this and #12

**Comment by randoum** [comment]
Thanks for your help.
autonumeric-rails v. 2.0.0.1 published on rubygems
